src/integrationTest/resources/tests/arm/properties/live/media-record.properties

```diff
@@ -0,0 +1,35 @@
+#OBJECT_TYPE
+#CASE_NUMBERS
+#FILE_TYPE
+#HEARING_DATE
+#CHECKSUM
+#TRANSCRIPT_REQUEST
+#TRANSCRIPT_TYPE
+#TRANSCRIPT_URGENCY
+#COMMENTS
+#PLACEHOLDER_DATE
+#CREATED_DATE_TIME
+#OBJECT_ID
+#PARENT_ID
+#CHANNEL
+#MAX_CHANNELS
+#UPLOADED_BY
+#START_DATE_TIME
+#END_DATE_TIME
+#COURTHOUSE
+#COURTROOM
+bf_001=OBJECT_TYPE
+bf_002=CASE_NUMBERS
+bf_003=FILE_TYPE
+bf_004=HEARING_DATE
+bf_005=CHECKSUM
+bf_010=CREATED_DATE_TIME
+bf_011=START_DATE_TIME
+bf_012=OBJECT_ID
+bf_013=PARENT_ID
+bf_014=CHANNEL
+bf_015=MAX_CHANNELS
+bf_017=END_DATE_TIME
+bf_019=COURTHOUSE
+bf_020=COURTROOM
+
```

Comment:
I'm assuming it's not a problem having this white space at the end, but just pointing it out since it's `live` and the others don't have it

Reply:
white space is not a problem in a properties file